### DISCUSSION TITLE
what are the podcast licenses?

### DISCUSSION THREAD

**Discussion by ductranit** [Podcastindex-org/podcast-namespace#583] [Q&A] [answered]
Hi there,
I just have few questions about the content license.
I have read about #177 but I don't see license info from API documentation.
What are the licenses for all podcast contents? 
Is it free to use for any purposes even commercial? 
And can I create a podcast client app for users to download & listen podcasts from podcastindex.org API?
Thanks!

**Comment by jamescridland** [comment, marked as answer]
There are three things to consider here:

1. The licence for the Podcast Index database. [This is a MIT licence here](https://github.com/Podcastindex-org/docs-api/blob/master/LICENSE).

2. The individual podcast RSS feeds. By publishing an open RSS feed, podcasters are inviting you to use (but not modify) the data held within the RSS feeds, for the use of promoting their podcasts.

3. The audio content. By publishing an open RSS feed, podcasters are inviting you to link to (but not modify or cache) the audio files.

The Podcast Index is clear about the use of its data; most podcast creators will also have those expectations about #2 and #3, though it's not as clear as you'd hope.

(Edited to add the MIT licence)

**Reply by ryan-lp** [reply]
> There are three things to consider here:
> 
> 1. The licence for the Podcast Index database. [This is a MIT licence here](https://github.com/Podcastindex-org/docs-api/blob/master/LICENSE).

It depends on how the database is obtained. The database dump is available for direct download from [here](https://github.com/Podcastindex-org/database) and that repository is made available under the MIT license.

However, the Podcast Index API is governed by its own Terms of Service [here](https://github.com/Podcastindex-org/legal/blob/main/TermsOfService.md). These terms prohibit building your own database from the API which is why I don't use it, and so you would be advised to check if you have any personal use cases that would also be prohibited. (Even if this prohibition wasn't intentional, it is legally still there in the terms of service, so it is binding).

**Comment by ductranit** [comment]
Great! thanks for letting me know.
